java/ReachSafety-JBMC.set

```diff
@@ -0,0 +1 @@
+jbmc-*/*.zip
```

Comment:
> This refers to ZIPs that are not there?

These zip files are built when you `make prepare`.

Reply:
>  I am not sure it makes sense to group tasks into categories like JBMC, JPF etc. 

A directory doesn't correspond to a category. The directories correspond to a source of origin of these benchmarks, which is how they are naturally grouped together. Any other suggestions are welcome.

Reply:
> But you could for now remove the .set files from the PR and open a discussion/issue on what category structure is desired for the Java tasks.

For now I suggest to have a single category because we have only 370 benchmarks.

Reply:
For directories, the current structure is fine, but you added `.set` files (which do correspond to categories) like `ReachSafety-JBMC.set`. A single category sounds good for me, too.

Reply:
Alright.

Reply:
Done